csr2transmart/mappers/ontology_mapper.py

```diff
@@ -70,8 +70,22 @@ def map_nodes(self, nodes: Sequence[OntologyConfigTreeNode], parent_node: TreeNo
                 parent_node.add_child(intermediate_node)
 
     def map(self, src_nodes: Sequence[OntologyConfigTreeNode]) -> Sequence[TreeNode]:
+        root_node: Optional[TreeNode] = None
+        top_node: Optional[TreeNode] = None
+        path = self.top_tree_node.split('\\')
+        for node in path:
+            if not node:
+                # Skip empty path elements
+                continue
+            current_node = TreeNode(node)
+            if top_node:
+                top_node.add_child(current_node)
+            else:
+                root_node = current_node
+            top_node = current_node
+        if not top_node:
+            raise OntologyConfigValidationException(f'Invalid top tree node: {self.top_tree_node}')
         date = datetime.datetime.now().strftime('%d-%m-%Y')
-        metadata = TreeNodeMetadata({'Load date': date})
-        top_node = TreeNode(self.top_tree_node, metadata)
+        top_node.metadata = TreeNodeMetadata({'Load date': date})
```

Comment:
shouldn't this be:
```suggestion
        root_node.metadata = TreeNodeMetadata({'Load date': date})
```

Reply:
No, I expect the metadata to appear in the top node. E.g., when using `\Public Studies\My Study\` as top node for a study, I expect the load date to appear on the study node, not on the `Public Studies` node (because it is different for each studies)

Reply:
ok, it makes sense!